web/satellite/src/components/accessGrants/CreateAccessModal.vue

```diff
@@ -0,0 +1,1028 @@
+                    >
+                    <div class="access-grant__modal-container__footer-container">
+                        <v-button
+                            :label="isPassphraseCopied ? 'Copied' : 'Copy to clipboard'"
+                            width="auto"
+                            height="50px"
+                            :is-transparent="isPassphraseCopied ? false : true"
+                            :is-white-green="isPassphraseCopied ? true : false"
+                            class="access-grant__modal-container__footer-container__copy-button"
+                            font-size="16px"
+                            :on-press="onCopyClick"
+                            :is-disabled="passphrase.length < 1"   
+                        />
+                        <v-button
+                            label="Download .txt"
+                            font-size="16px"
+                            width="auto"
+                            height="50px"
+                            class="access-grant__modal-container__footer-container__download-button"
+                            :is-green-white="isPassphraseDownloaded ? true : false"
+                            :on-press="downloadText"
+                            :is-disabled="passphrase.length < 1"
+                        />
+                    </div>
+                    <div v-if="isPassphraseDownloaded || isPassphraseCopied" :class="`access-grant__modal-container__acknowledgement-container ${acknowledgementCheck ? 'blue-background' : ''}`">
+                        <input
+                            v-model="acknowledgementCheck"
+                            type="checkbox"
+                            class="access-grant__modal-container__acknowledgement-container__check"
+                        >
+                        <div class="access-grant__modal-container__acknowledgement-container__text">I understand that Storj does not know or store my encryption passphrase. If I lose it, I won't be able to recover files.</div>
+                    </div>
+                    <div
+                        v-if="isPassphraseDownloaded || isPassphraseCopied"
+                        class="access-grant__modal-container__acknowledgement-buttons"
+                    >
+                        <v-button
+                            label="Back"
+                            width="auto"
+                            height="50px"
+                            :is-transparent="true"
+                            class="access-grant__modal-container__footer-container__copy-button"
+                            font-size="16px"
+                            :on-press="backAction"
+                        />
+                        <v-button
+                            label="Create my Access ⟶"
+                            font-size="16px"
+                            width="auto"
+                            height="50px"
```

Comment:
Styling belongs into css.

Reply:
Hi Egon, I can try to make the change in my code, however this button is used a countless number of times throughout the application with height and width being set as properties. If we wanted to remove it from the component as a whole, fixing all of the other buttons utilizing this would be a much bigger effort.